examples/kuesa/tonemapping/assets.qrc

```diff
@@ -0,0 +1,25 @@
+<RCC>
+    <qresource prefix="/">
+        <file>../assets/envmaps/wobbly-bridge/CREDITS.txt</file>
+        <file>../assets/envmaps/wobbly-bridge/wobbly_bridge_4k_cube_irradiance.dds</file>
```

Comment:
these envmap files are missing. Why do you need new ones and not use the ones that were already there?